app/shared/productTypes.tsx

```diff
@@ -94,16 +103,35 @@ export interface HolidayStopFlowProperties {
   };
 }
 
+export interface DeliveryProblemType {
+  label: string;
+  messageIsMandatory: boolean;
+}
+
+export const commonDeliveryProblemTypes: DeliveryProblemType[] = [
+  { label: "Damaged Paper", messageIsMandatory: true },
+  { label: "Delivered Despite Holiday", messageIsMandatory: false },
+  { label: "No Delivery", messageIsMandatory: false },
+  { label: "Other", messageIsMandatory: true }
+];
+
+interface DeliveryRecordsProperties {
+  showDeliveryInstructions?: true;
+  numberOfProblemRecordsToShow: number;
```

Comment:
I can't actually see this being used to limit the items available for selection in 'report problem' mode?